src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -92,5 +92,10 @@ public class StorageConfiguration : ConfigurationCategory
         [DefaultValue(null)]
         [ConfigurationEntry("Storage.Dangerous.IgnoreInvalidJournalErrors", ConfigurationEntryScope.ServerWideOnly)]
         public bool? IgnoreInvalidJournalErrors { get; set; }
+
+        [Description("EXPERT: Skip checksum validation on database loading process (applicable only for ARM 32/64)")]
```

Comment:
Why applicable to ARM only? Don't we want it to be generally available?

Reply:
On my request.

Reply:
What is the reasoning?